addons/udes_stock/README.md

```diff
@@ -151,12 +152,18 @@ This is essentially a collection of products that are to be moved from one locat
 | _create_batch | Creates a batch |
 | create_picking | Create and return a picking for the given picking_type. For multiple pickings a list of lists of dicts of product_info should be passed, and pickings with the same picking_type and other kwargs are the same. The kwargs are applied to pickings, not moves. If needed, the moves can be created outside of create_pickings with _create_moves. |
 | _create_move | Create and return move(s) for the given move_values. Should be used in conjunction with _prepare_move to obtain move_values |
+| get_is_empty_pickings | Find empty pickings that will be deleted|
```

Comment:
Think this method is `get_empty_pickings`? The description doesn't need to be about deletes, might be better to know it is a gen and uses u_is_empty not a check on the mls/moves?

Reply:
Changed, please review.